krypto-api/publish.gradle

```diff
@@ -32,74 +40,68 @@ if ( ( branch.equals("develop") || branch.equals("master") )
         sign configurations.pom
     }
 
-    ext."signing.keyId" = 'FDEB5A45'
-    ext."signing.secretKeyRingFile" = "$System.env.bamboo_JAR_SIGNING_KEYRING" 
-    ext."signing.password" = "$System.env.bamboo_JAR_SIGNING_PASSWORD"
-
-    if ( branch.equals("develop") ) {
-        publishing {
-            repositories {
-                maven { 
-                    url snapshotUrl
-                    credentials {
-                        username = "$System.env.bamboo_SONATYPE_USERNAME"
-                        password = "$System.env.bamboo_SONATYPE_PASSWORD"
-                    }
+    publishing {
+        repositories {
+            maven {
+                url branch.equals("master") ? releaseUrl : snapshotUrl
+                credentials {
+                    username = "$System.env.bamboo_SONATYPE_USERNAME"
+                    password = "$System.env.bamboo_SONATYPE_PASSWORD"
                 }
             }
-
-            publications {
-                mavenJava(MavenPublication) {
-                    from components.java
-
-                    artifact sourcesJar
-
-                    pom.withXml {
```

Comment:
It's just been moved. I'd duplicated everything in a second conditional based on whether the branch was master or develop. This is basically identical to rhizome, iris and kodex now.